pathfinder/pathfinding_service.py

```diff
@@ -234,6 +236,34 @@ def on_balance_proof_message(self, balance_proof: BalanceProof):
                 balance_proof.locked_amount,
             )
 
+    def on_paths_request_message(self, paths_request: PathsRequest):
+        self._check_chain_id(paths_request.chain_id)
```

Comment:
you should catch the exception, otherwise any message with a wrong chain id will kill the server